tuf/client_rework/updater_rework.py

```diff
@@ -35,27 +36,32 @@ class Updater:
     """
     Provides a class that can download target files securely.
 
-    Attributes:
-        metadata:
-
-        repository_name:
-
-        mirrors:
-
-        fetcher:
-
-        consistent_snapshot:
+    TODO
```

Comment:
Is this covered by #1385?

Reply:
Yeah that was the idea: The public API will change a bit as we start using the MetadataBundle so let's document after that